larky/src/main/java/com/verygood/security/larky/modules/VaultModule.java

```diff
@@ -188,17 +194,90 @@ public void delete(Object value, Object storage) throws EvalException {
         vault.delete(value, storage);
     }
 
+    @StarlarkMethod(
+        name = "sign",
+        doc = "signs a payload with given keyId",
+        parameters = {
+            @Param(
+                name = "keyId",
```

Comment:
should this be called `keyArn`? i think id and arn are distinct in aws platform.

https://docs.aws.amazon.com/kms/latest/developerguide/find-cmk-id-arn.html

Reply:
Are we exposing AWS internals too much? I understand the lack of abstraction, but would this be a breaking change if someone decided to use a different key provider?

Reply:
we can keep it as keyId to keep it as generic as possible, but if we end up supporting HSMs outside KMS, things will get really complicated really fast

Reply:
yeah, we can add a `provider` parameter later. i see your point on `keyId`.

Reply:
Every place we use this in the current code base, we use `...KEY_ID`